docs/getting-started.md

```diff
@@ -36,6 +36,7 @@ Go to [releases](https://github.com/FS-Online/Formula-Student-Driverless-Simulat
 Unzip it to anywhere on your computer and launch FSDS.exe.
 A window with a car should popup!
 Try driving the car around using the arrowkeys.
+If you get a black screen with some buttons, make sure the folder with the binary is in your user folder (Windows: `C:\Users\username\Formula-Student-Driverless-Simulator`, Linux: `~/Formula-Student-Driverless-Simulator`)
```

Comment:
I believe the binary should work placed anywhere. What kind of black screen with some buttons did you encounter that made you conclude that the binary has to be in the user folder?

Reply:
I was referring to issue #224, might be fixed by now